01-start/03-begin.html.md

```diff
@@ -565,7 +560,7 @@ As you now have all the tools and knowledge required to be able to create the bl
 
 Congratulations! You now possess all the foundations required to be able to write amazing and powerful web applications like those already in our [Showcase](/docpad/showcase). To recap, you now know how to:
 
-- write documents in any language, markup, pre-processor, templating engine, whatever you wish, by installing the necessary plugin for it and changing the extensions of the document
+- write `render` files in any language, markup, pre-processor, templating engine, whatever you wish, by installing the necessary plugin for it and changing the extensions of the document
```

Comment:
Yeah, I ran in to a couple of issues with the terminology, especially multiple meanings of the word "document," a word that's very hard to use consistently.

I settled, for the moment, for the term "render file" for files under the `render/` directory that are input to the render process and not layouts. I'm not particularly pleased with it, however, and would love to see something better.

My sense, especially in light of the internal class/model descriptions above, is that a "document" is actually the output of the rendering process. Thus, each render file is really the specification for a document, giving the name, location and (some of the) contents of a document; the actual document is produced in the `out/` directory by a rendering process combining that with layout information. This implies that the previous name of the `render/` directory, `documents/` was really very wrong, so wrong that I'd suggest we prioritize getting away from it completely.

That would involve, for a start, fixing the init process to actually create directories with the proper names, and going through and fixing all the skeletons. Ideally, we should also provide some help for migrating existing projects so as to erase this confusion from the face of the earth.